blocklylib-core/src/main/java/com/google/blockly/android/ui/VirtualWorkspaceView.java

```diff
@@ -106,7 +107,6 @@ public void onFinishInflate() {
         setHorizontalScrollBarEnabled(mScrollable);
         setVerticalScrollBarEnabled(mScrollable);
 
-        mScaleGestureDetector = new ScaleGestureDetector(getContext(), new ScaleGestureListener());
```

Comment:
Revert this.  Instead of removing the gesture detector, use the zoom behavior (above) to determine what to update within onScale().